src/alfasim_sdk/_internal/cli.py

```diff
@@ -111,121 +116,33 @@ def template(dst, caption, plugin_id, author_name, author_email):
     python_folder.mkdir()
     Path(python_folder / f"{plugin_id}.py").touch()
 
+    # remove compile.py created by hookman
+    compile_file = dst / plugin_id / "compile.py"
+    if compile_file.exists():
```

Comment:
```suggestion
    if compile_file.is_file():
```

General advice: prefer to be specific if you mean a file or a directory.

Reply:
Nice catch, thanks, I've added your suggestion.